daheinze-oscheer/hello/hello.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+const spanish = "Spanish"
+const french = "French"
+const englishHelloPrefix = "Hello, "
+const spanishHelloPrefix = "Hola, "
+const frenchHelloPrefix = "Bonjour, "
+
+func Hello(name string, language string) string {
```

Comment:
Each function should have a comment in the form of: <funcname> <verb> ....

```suggestion
// Hello returns a Hello World greeting for multiple languages
func Hello(name string, language string) string {
```